modules/machineset-yaml-aws.adoc

```diff
@@ -28,6 +28,8 @@ spec:
   selector:
     matchLabels:
       machine.openshift.io/cluster-api-cluster: <infrastructureID> <1>
+      machine.openshift.io/cluster-api-machine-role: <role> <3>
+      machine.openshift.io/cluster-api-machine-type: <role> <3>
```

Comment:
`<3>` should be `<2>` for above 2 lines

Reply:
Actually no need to update here, if we want the machines to be labeled, we should update 

```
spec:
  template:
    metadata:
      labels:
```